amxmodx/CTask.cpp

```diff
@@ -269,8 +269,10 @@ bool CTaskMngr::taskExists(int iId, AMX *pAmx)
 
 void CTaskMngr::startFrame()
 {
-	for (auto &task : m_Tasks)
+	for(auto i = m_Tasks.length() - 1; i != 0; i--)
```

Comment:
Actually, we want to keep the order that the tasks have been created. You have plugins creating multiple tasks at once, this will break this behavior. 

So, we need more looping from 0 to saved size:
```PAWN
auto size = static_cast<int>(m_Tasks.length());
for (auto i = 0; i < size; ++i) 
{
`